stripe/src/main/java/com/stripe/android/model/PaymentIntent.java

```diff
@@ -32,26 +32,30 @@
 public class PaymentIntent extends StripeJsonModel {
     private static final String VALUE_PAYMENT_INTENT = "payment_intent";
 
-    private static final String FIELD_ID = "id";
-    private static final String FIELD_OBJECT = "object";
-    private static final String FIELD_ALLOWED_SOURCE_TYPES = "allowed_source_types";
-    private static final String FIELD_AMOUNT = "amount";
-    private static final String FIELD_CREATED = "created";
-    private static final String FIELD_CANCELED = "canceled_at";
-    private static final String FIELD_CAPTURE_METHOD = "capture_method";
-    private static final String FIELD_CLIENT_SECRET = "client_secret";
-    private static final String FIELD_CONFIRMATION_METHOD = "confirmation_method";
-    private static final String FIELD_CURRENCY = "currency";
-    private static final String FIELD_DESCRIPTION = "description";
-    private static final String FIELD_LIVEMODE = "livemode";
-    private static final String FIELD_NEXT_SOURCE_ACTION = "next_source_action";
-    private static final String FIELD_RECEIPT_EMAIL = "receipt_email";
-    private static final String FIELD_SOURCE = "source";
-    private static final String FIELD_STATUS = "status";
+    static final String FIELD_ID = "id";
+    static final String FIELD_OBJECT = "object";
+    static final String FIELD_ALLOWED_SOURCE_TYPES = "allowed_source_types";
```

Comment:
i think, for these changes, we should be removing the legacy field names and cutting fully over to the new field names. this would mean removing `allowed_source_types`, and `next_source_action`. wdyt?

Reply:
Let's say an integrating app calls `PaymentIntent#getNextSourceAction()`. Currently that method returns the value of `"next_source_action"`. Would we be able to simply change it to return `PaymentIntent#getNextAction()` and everything would "just work"?